src/modules/sessionManager.js

```diff
@@ -101,15 +102,34 @@ class SessionManager {
     /**
      * Convert a fetch response to json when the result was valid.
      *
-     * @param {*} response
+     * @param {Response} response
      * @returns json as object or a rejected Promise when the response status was not 2xx or the Content-Type was not application/json
      */
-    toJson = (response) => {
+    static toJson = (response) => {
         if (response.status < 200 || response.status >= 300 || response.headers.get('Content-Type').indexOf('application/json') !== 0) {
             return Promise.reject(response);
         }
         return response.json();
     };
+
+    /**
+     * Parse the UUID from the Location header after an object is successfully created (project, singlefile, task).
+     *
+     * @param {Response} response The window.fetch response
+     * @returns A promise with the id
+     */
+    static parseLocationId = (response) => {
+        if (response.status === 201) {
+            const location = response.headers.get('Location');
+            if (location) {
+                const id = location.match(/^\/[a-z]+\/([a-z0-9-]+)$/);
+                if (id && id.length === 2) {
```

Comment:
So we expect two `UUID`s in the `Location` header, but we pick only the first one? Maybe this would warrant a line comment (perhaps with an example of how the `Location` header typically looks).

Reply:
No, we expect `/projects/12343-12332343-343343-334343` or `/tasks/{any uuid here}`